toolbox

```diff
@@ -1596,6 +1596,7 @@ run()
             --tty \
             --user "$USER" \
             --workdir "$PWD" \
+            --detach-keys="" \
```

Comment:
Let's not touch the shell script here because doing the version check would be too cumbersome than it's actually worth. So, not changing it at all seems better than a half-hearted change.

Reply:
Done.